app/utilities/featureToggleParser/featureToggleParser.factory.spec.js

```diff
@@ -2,8 +2,303 @@
 
 describe('featureToggleParser', function () {
 
-  it('TBD',function() {
+  beforeEach(module('myApp.utilities'));
 
+  var featureToggleParser;
+  
+  beforeEach(inject(function(_featureToggleParser_) {
+  
+      featureToggleParser = _featureToggleParser_;
+  
+  }));
+  
+  it('should featureToggleParser be defined', function() {
+  
+      expect(featureToggleParser).toBeDefined();
+  
+  });
+  
+  it('should featureToggleParser.parseConfigObject() be defined', function() {
+  
+      expect(featureToggleParser.parseConfigObject).toBeDefined();
+  
+  });
+  
+  it('should featureToggleParser.parseToggleObject() be defined', function() {
+  
+      expect(featureToggleParser.parseToggleObject).toBeDefined();
+  
+  });
+  
+  describe('parseConfigObject()', function() {
+  
+      var result, newConfig;
+  
+      it("should return false for enableDates when the parseConfigObject is called with enableDates as ['false', 'off', '0', 'no']", function() {
+  
+          newConfig = {
+              "enableDates": 'no'
+          };
+  
+          result = featureToggleParser.parseConfigObject(newConfig);
+  
+          expect(result.enableDates).toBeFalsy();
+  
+      });
+  
+      it("should return false for enableDates when the parseConfigObject is called with enableDates as random String", function() {
```

Comment:
good call here, demonstrates that the above negative value test is inconclusive since any unknown value is translated to false.